i18n/fi_FI.csv

```diff
@@ -110,3 +110,5 @@ Hello,Hei
 "Select Card","Valitse kortti"
 "Unable to change card","Korttia ei voitu vaihtaa"
 "Card for subscription changed successfully","Kortin vaihto toistuvaan tilaukseen onnistui"
+"Subscription couldn't be canceled","Tilausta ei voitu peruuttaa"
+"Subscription has been canceled correctly","Tilaus on peruutettu oikein"
```

Comment:
While the translation is alright it is not something native speaker would say.
You could shorten it to "Tilaus on peruutettu" 
The "oikein" (translated 'correctly') is a bit redundant in finnish.

Reply:
Done